src/main/java/gov/nasa/pds/tools/validate/rule/pds4/TableDataContentValidationRule.java

```diff
@@ -133,11 +174,136 @@ public boolean isApplicable(String location) {
+                LOG.error("ASCII String related fields should not contain '+' symbol in field_format [" + oneFieldFormat + "] in field_number " + fieldNumberList.get(indexToList));
+                addTableProblem(ExceptionType.ERROR, 
+                    ProblemType.TABLE_FILE_READ_ERROR,
+                    "ASCII String related fields should not contain '+' symbol in field_format [" + oneFieldFormat + "] in field_number " + fieldNumberList.get(indexToList),
+                    getTarget(),
+                    -1,
+                    -1);
+            } 
+       }
+       indexToList += 1;
+    }
+  }
+
+  private void validateAsciiNumberFieldsFormat(List<String> fieldFormatList, List<String> fieldTypeList, List<String> fieldNumberList) {
+    // ASCII Number related fields should not have '-' in the format.
+
+    int indexToList = 0;
+    // Loop through all elements fieldFormatList.  If field is in ASCII_NUMBER_TYPE_LIST, check if prohibited '-' is in the format.
+    for (String oneFieldFormat : fieldFormatList) {
+        //LOG.debug("validateAsciiNumberFieldsFormat:indexToList,oneFieldFormat,fieldTypeList.get(indexToList)  {},{},{}",indexToList,oneFieldFormat,fieldTypeList.get(indexToList));
+        // Check for prohibited minus symbol '-' in format field : %-2d
+        if (this.ASCII_NUMBER_TYPE_LIST.contains(fieldTypeList.get(indexToList).toUpperCase())) {
+            if (oneFieldFormat.contains("%-")) {
+                LOG.error("ASCII Number related fields should not contain '-' symbol in field_format [" + oneFieldFormat + "] in field_number " + fieldNumberList.get(indexToList));
+                addTableProblem(ExceptionType.ERROR, 
+                    ProblemType.TABLE_FILE_READ_ERROR,
+                    "ASCII Number related fields should not contain '-' symbol in field_format [" + oneFieldFormat + "] in field_number " + fieldNumberList.get(indexToList),
+                    getTarget(),
+                    -1,
+                    -1);
+            }
+       }
+       indexToList += 1;
+    }
+  }
+
+  private void validateFieldFormats() {
+     // The field 'field_format' need to checked for conformant to standard:
+     // Example of valid field_format for ASCII_String (left justified):
+     //
+     //     <data_type>ASCII_String</data_type>
+     //     <field_length unit="byte">26</field_length>
+     //     <field_format>%-26s</field_format>
+     //
+     // Example of an invalid field_format for ASCII_String (right justified):
+     //          <data_type>ASCII_String</data_type>
+     //          <field_length unit="byte">8</field_length>
+     //          <field_format>%+8s</field_format>
+     //          <description>Number of the asteroid.</description>
+     int numFields = 0;
+
+     try {
+        XMLExtractor extractor = new XMLExtractor(getTarget());
+        TinyNodeImpl productObservationalNode = extractor.getNodeFromDoc(this.PRODUCT_OBSERVATIONAL);
```

Comment:
looks like static variables are being accessed with `this` throughout. can we just do a quick scrub of this new code to make sure we are accessing this properly.